package.json

```diff
@@ -1,24 +1,12 @@
 {
   "name": "smart-app-banner",
-  "version": "1.3.0",
-  "description": "Smart banner for Android or iOS",
-  "main": "index.js",
-  "scripts": {
-    "build": "browserify index.js -s SmartBanner | ccjs - > smart-app-banner.js"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/kudago/smart-app-banner.git"
+  "version": "%version%",
```

Comment:
that's just for the `master` branch, if new version would be released only content in `version.txt` must be changed, after accepting PR travis will build new version, if succeeded, `%version%` in files is replaced it that one in `version.txt` and created new tag with that version.